app/assets/css/Semantic-UI-CSS-master/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
Consegui sim, essa pasta faltou excluir mesmo. 




Atenciosamente.Victor H.P. Cunha | Programador WEB | Trained Teacher Microsoft
Ebecom | Meta Cursos
Rua Engenheiro Antônio Rebouças, 460 – Santa Cruz | Cep: 85015-410Fone: 42 30355240 | 42 99701365 | http://www.ebecomgroup.com.br 

    Em Segunda-feira, 9 de Maio de 2016 21:56, Fabio Leandro Janiszevski <notifications@github.com> escreveu:
 

 In app/assets/css/Semantic-UI-CSS-master/LICENSE:> @@ -0,0 +1,22 @@
> +The MIT License (MIT)
@xtiurti Não conseguiu utilizar os arquivo do semantic da própria gem?—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub